src/java/components/hu.bme.mit.inf.modes3.components.dashboard/src/main/webapp/public_html/js/config.js

```diff
@@ -278,102 +248,108 @@ var settings = {
 		"t06", "t06-div", "t06-str"
 	],
 	locomotives : [
-	
-	//ide sok vonat
-		[ 9, {
+		[ "Taurus", {
 			name : "Taurus MÁV 1047",
 			address : 9,
 			image : "taurus-mav-1047.jpg",
 			preview: "taurus-mav-1047-preview.jpg",
 			svgGroup : "taurus",
 		}],
 		
-		[ 10, {
+		[ "SNCF", {
 			name : "SNCF 68000",
 			address : 10,
 			image : "sncf-68000.jpg",
 			preview: "sncf-68000-preview.jpg",
 			svgGroup : 2,
 		}],
 		
-		[ 11, {
-	        name: "BR-204",
+		[ "BR294", {
+	        name: "BR-294",
 	        address: 8,
 	        image: "br-204.jpg",
 			preview: "br-204-preview-2.jpg",
 	        svgGroup: 3
 	    }],
 		
-		[ 12, {
-			name : "NOHAB MÁV M61 #1",
-			address : 12,
-			image : "nohab.jpg",
-			preview: "nohab-preview.jpg",
-			svgGroup : "nohab",
-		}],
-		
-		[ 13, {
-			name : "Siemens Desiro",
-			address : 13,
-			image : "vagottdesiro.jpg",
-			preview: "desiro-preview.jpg",
-			svgGroup : "desiro2",
-		}],
-		
-		[ 14, {
-			name : "MÁV V43",
-			address : 14,
-			image : "szili.jpg",
-			preview: "szili-preview.jpg",
-			svgGroup : "szili",
-		}],
-		
-		[ 15, {
-			name : "NOHAB MÁV M61 #2",
-			address : 15,
-			image : "nohab.jpg",
-			preview: "nohab-preview.jpg",
-			svgGroup : "nohab",
-		}],
-		
-		[ 16, {
-			name : "NOHAB MÁV M61 #3",
-			address : 16,
-			image : "nohab.jpg",
-			preview: "nohab-preview.jpg",
-			svgGroup : "nohab",
-		}],
-		[ 17, {
-			name : "NOHAB MÁV M61 #4",
-			address : 17,
-			image : "nohab.jpg",
-			preview: "nohab-preview.jpg",
-			svgGroup : "nohab",
-		}],
-		[ 18, {
-			name : "NOHAB MÁV M61 #5",
-			address : 18,
-			image : "nohab.jpg",
-			preview: "nohab-preview.jpg",
-			svgGroup : "nohab",
-		}],
-		[ 19, {
-			name : "NOHAB MÁV M61 #6",
-			address : 19,
-			image : "nohab.jpg",
-			preview: "nohab-preview.jpg",
-			svgGroup : "nohab",
-		}],
-		[ 20, {
-			name : "NOHAB MÁV M61 #7",
-			address : 20,
-			image : "nohab.jpg",
-			preview: "nohab-preview.jpg",
-			svgGroup : "nohab",
-		}]
+//		[ 12, {
```

Comment:
Instead of putting these lines into a comment block, they could be removed.

Reply:
This one is already removed.